### DISCUSSION TITLE
the resolutions in major and minor, background play and range

### DISCUSSION THREAD

**Discussion by xyzxyyz** [ShacharHarshuv/open-ear#205] [Ideas]
regarding automatic play, resolutions, background play and range
hi, since the functional ear trainer app doesn’t offer this functionality on ios at all, neither paid nor free, I was moved to try out your opeanear app, of course, at first it seemed to good to be true. it’s amazing, with the chord functions etc.
but first, there’s a bug. settings: automatic answer the question, automatic go to the next question. play resolution.
the notes 4#, 6b, 7b don‘t give a error, but with 2b and 3b it does (unexpected empty resolution). so all notes work except 2b and 3b.

and there’s something about the resolutions 
the resolution of 6b plays 6b, 7b and 1, with the major cadence(!) and minor cadence. in the other f.e.t. app the resolution is 6b 6 7 1 in major, or 6b 7b 1 when in minor mode. similarly the resolution of 6 in minor mode would be 6, 7b and 1.
but the resolution of 6 in major mode in the open ear app is 6, 7b(!), 1.

in the open ear app the resolution also depends on which notes are activated.
for example, with 6b in the openear app, the cadence would be 6b 7b and 1. 
if I activated the note 6 also, then it changes to 6b 6 7b and 1. strange!

it’s just my recommendation that you make the resolutions dependent on which scale is used. if in major, then the major resolution, if in minor, the minor resolution (i advise you to use the same resolutions like in the f.e.t. app)
and also make the resolution independent on which notes are activated, so the resolution is always the same.
it would also be great if the sound could be played in the background of ios and if the range could be high and low and not only high or low or middle.

thank your for building the app, i know it’s all voluntary! i would help if i could program. if it’s too much work you could make it paid and i‘d pay for the app in the app store.

**Comment by ShacharHarshuv** [comment]
> since the functional ear trainer app doesn’t offer this functionality on ios at all, neither paid nor free

Is it possible the app store version is just out of date? It's harder for me to deploy to app store so that might be the case. 

> it’s just my recommendation that you make the resolutions dependent on which scale is used. if in major, then the major resolution,

Can you make a clearer list of scenario + expected results? Your previous message is a bit hard to understand.
